examples/apps/quick_tap.py

```diff
@@ -0,0 +1,466 @@
+
+        Args:
+            round_start_time (Time): time stamp set in QuickTapGame to calculate players' tap_times
+        '''
+        self.elapsed_tap_time = time.time() - round_start_time
+        self.has_tapped = True
+
+
+class CozmoQuickTapPlayer(QuickTapPlayer):
+    '''Cozmo-specific Quick Tap player logic, with a reference to the actual Cozmo robot.
+        
+    Args:
+        robot (cozmo.robot.Robot): passed in from the QuickTapGame class
+    '''
+    def __init__(self, robot: cozmo.robot.Robot):
+        super().__init__()
+        self.robot = robot
+        self.name = 'Cozmo'
+
+    async def select_cube(self):
+        '''Cozmo looks for a cube, drives to it, and taps it.'''
+        self.cube = await self.robot.world.wait_for_observed_light_cube()
+        self.cube.set_lights(cozmo.lights.white_light)
+        await asyncio.sleep(2)
+        self.cube.start_light_chaser(0.5)
+        await self.robot.set_lift_height(1.0).wait_for_completed()
+        await self.robot.go_to_object(self.cube, distance_mm(40)).wait_for_completed()
+        await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapTap).wait_for_completed()
+        self.cube.stop_light_chaser()
+        self.cube.set_lights(green_light)
+
+    async def determine_move(self, buzzer_display_type):
+        '''Cozmo chooses a move based on the probabilities above.
+
+        Args:
+            buzzer_display_time (string): the display of the buzzers, either RED, SAME, or DIFFERENT
+        '''
+        await self.hesitate()
+        probability_correct = random.random()
+        if probability_correct < RATE_COZMO_ACCURACY:
+            if buzzer_display_type == MAKE_BUZZERS_SAME_COLORS:
+                await self.tap()
+            else:
+                await self.fail_to_tap()
+        else:
+            if buzzer_display_type == MAKE_BUZZERS_RED or buzzer_display_type == MAKE_BUZZERS_DIFFERENT_COLORS:
+                await self.tap()
+            else:
+                await self.fail_to_tap()
+
+    async def hesitate(self):
+        '''Cozmo waits between 0 and 0.5 seconds'''
+        delay = random.random() * .5
+        await asyncio.sleep(delay)
+
+    async def tap(self):
+        '''Calls Cozmo's tap animation.'''
+        await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapTap).wait_for_completed()
+
+    async def fail_to_tap(self):
+        '''Randomly calls either Cozmo's fakeout tap animation or his idle animation.'''
+        probability_fakeout = random.random()
+        if probability_fakeout < 0.5:
+            await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapFakeout).wait_for_completed()
+        else:
+            await self.robot.play_anim_trigger(cozmo.anim.Triggers.OnSpeedtapIdle).wait_for_completed()
+
+
+rainbow_colors = [blue_light, red_light, green_light, yellow_light]
+
+class BlinkyCube(cozmo.objects.LightCube):
+    '''Same as a normal cube, plus extra methods specific to Quick Tap.
+
```

Comment:
I would remove the Args listing here instead of saying "same arguments as cozmo.objects.LightCube".